calling-apis/intro-to-asynchronous-programming.md

```diff
@@ -0,0 +1,136 @@
+# Intro to Asynchronous Programming
+
+## Learning Goals
+
+- Define asynchronous programming
+- Compare synchronous and asynchronous programming
+
+## Introduction: A Pizza Metaphor
+
+It's Friday, and Mica is exhausted and spending the evening at home. It's dinner time, and they have no food in their freezer nor fridge. They aren't going to go grocery shopping, and nobody will make dinner for them tonight.
+
+Instead, they'll make a phone call to their favorite restaurant (who's not on an app) and order dinner for themself for delivery.
+
+There's a series of things that happen between the moment Mica makes the order all the way to the point where they take the first bite.
+
+In pain-staking detail, let's outline all the actions Mica would do, beginning with picking up the phone, and ending with eating the meal.
```

Comment:
no hyphenation (painstaking derives from "taking pains" rather than "staking pain")

```suggestion
In painstaking detail, let's outline all the actions Mica would do, beginning with picking up the phone, and ending with eating the meal.
```